### ISSUE TITLE
How to merge 2 households if both have common usagepoint

### ISSUE THREAD

**Issue by taha-k** [CIVIS-project/YouPower#81]
This is a special case, where 2 households already exists and members of these households try to connect with the same usagepoint. Then we need a way to merge both the households into one and transferring all the members,actions etc into one

**Comment by FruitieX** [comment]
I don't think we will provide an option for merging households, and I guess the app will refuse to add a usage point to a household if it's already been added to another household. If the user moves to a new house, then they should ask someone in the house for an invite, and once they accept the invite they will be transferred to the new household.

**Comment by FruitieX** [comment]
Need to delete the old household, when the last person leaves it.
